mlrun/feature_store/api.py

```diff
@@ -393,7 +393,7 @@ def ingest(
         raise mlrun.errors.MLRunInvalidArgumentError(
             "featureset.spec.engine must be set to 'spark' to ingest with spark"
         )
-    if featureset.spec.engine == "spark":
+    if featureset.spec.engine == "spark" and run_config is not None:
```

Comment:
But now it'll just continue to a non-spark ingest.
It has to be added in the if isinstance  in the next line.